src/loggers/database-logger.php

```diff
@@ -0,0 +1,118 @@
+<?php
+/**
+ * A simple logger to allow query and memory usage logging.
+ *
+ * @package Yoast\WP\Free\Initializers
+ */
+
+namespace Yoast\WP\Free\Loggers;
+
+use YoastSEO_Vendor\ORM;
+
+/**
+ * Class Log_Helper
+ */
+class Database_Logger {
```

Comment:
I'd turn this class into an integration.

Move the `__construct` to `register_hooks`.
Create a new Conditional `Development_Conditional` that `is_met` when `\WPSEO_Utils::is_development_mode()`,

Reply:
Done.